packages/esbuild/helpers.bzl

```diff
@@ -0,0 +1,108 @@
+"""
+Utility helper functions for the esbuild rule
+"""
+
+load("@bazel_skylib//lib:paths.bzl", "paths")
```

Comment:
This is the first time we make users depend on skylib, but it doesn't seem necessary here. We aren't using a provider symbol that has to be shared, just need a bit of logic?
I think we should vendor the one function we need into our third_party